aws/sdk/Cargo.lock

```diff
@@ -443,6 +443,30 @@ dependencies = [
  "paste",
 ]
 
+[[package]]
+name = "aws-runtime"
+version = "1.2.2"
```

Comment:
Do we know why `aws-runtime` with its source crates.io-index has been downgraded to `1.2.2` from `1.3.0`?

Reply:
Good catch! I think (although crates.io doesn't seem to preserve timestamps so hard to say for sure) that when I actually generated these lockfiles that 1.3.0 might not have been published yet since it is only 3 days old. Took me a couple days to get through all the e2e-testing with the unpinned versions. I regenerated and that bit disappeared so should be good after another CI run.